src/cbexigen/tools_config.py

```diff
@@ -184,3 +185,55 @@ def process_config_parameters():
     # c_replace_chars (replace with underscore)
     if hasattr(config_module, 'c_replace_chars'):
         CONFIG_PARAMS['c_replace_chars'] = config_module.c_replace_chars
+
+
+ISO2_SCHEMAS_URL = "https://standards.iso.org/iso/15118/-2/ed-2/en/"
+ISO20_SCHEMAS_URL = "https://standards.iso.org/iso/15118/-20/ed-1/en/"
```

Comment:
Can we add DIN as well? The default config requires this.

I have found some peculiar DIN XSDs which drop the generator into an endless loop (probably because they incorrectly reference ISO schemas), but I have found one proper location.

Reply:
Since there is no official DIN or ISO place where you can download the DIN schemas, I would not add DIN.

Reply:
Okay, I agree.

I will propose a documentation change.